src/components/layout/layout.tsx

```diff
@@ -187,14 +187,12 @@ const Layout = ({
             if (allow_tracking) {
                 window.onload = () => {
                     window.setTimeout(() => {
```

Comment:
Hi @yashim-deriv  just a question, do we need the setTimeout here, as it already has the dependencies?

Reply:
This is from an older implementation. I think the reasoning is to fire this event after the `page_load` event defined in the `gatsby-browser.js`